samples/openthread/coap_client/README.rst

```diff
@@ -46,6 +49,14 @@ Button 2:
   Pressing results in sending a multicast LIGHT_ON or a multicast LIGHT_OFF message (alternatively) to the ``/light`` resource on server nodes.
   Sending this kind of message instead of a LIGHT_TOGGLE one allows to synchronize the state of the LEDs on several server nodes at once.
 
+Button 3:
```

Comment:
Is this Button 3 usage is valid for all samples that support power consumption mode switching? If so, this should also be mentioned in the user guide.

Reply:
Currently it is valid only for this sample when is build as MTD with SED support.

Reply:
OK, let's keep it here.